.travis.yml

```diff
@@ -6,10 +6,6 @@ env:
 matrix:
   fast_finish: true
   include:
-    - env: DM=~6.0
-      php: 5.3
-    - env: DM=~6.0
```

Comment:
So now I moved those version number down (if I understood you correctly). I've also added an extra pair DM 6.0, PHP 5.5. to have a PHP 5.5 build that is not allowed to fail.
I don't know if such multiple builds per language version are something that is wanted here, though.